### DISCUSSION TITLE
Who's hiring (Feb 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#45533] [Show and tell]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #46890.

*If you're looking for last month's discussion, see #44541. Several companies [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result of it!*

**Comment by leggetter** [comment]
Tigris Data | Senior Developer Advocate | Remote | Full-time | $100k - $155k base |  [Senior Developer Advocate @ Tigris Data](https://www.notion.so/tigrisdata/Senior-Developer-Advocate-3c13a312b52f4f5abb6cfef0c6522b63)

Tigris Data is hiring a Senior Developer Advocate to join our DX team. Our founders previously built the data developer platform at Uber; they've now founded [Tigris Data](https://www.tigrisdata.com) to build Tigris, an [open-source Data Developer Platform](https://github.com/tigrisdata/tigris). We're a seed-stage company, so we all do a bit of everything. But the DX team's day-to-day focus is example code, content (docs, blog, videos), and community.

I joined Tigris Data in December, and I can honestly say that I'm enjoying this role more than I have any other in years (I've previously worked at Pusher, Nexmo/Vonage, PostHog, and Ably). The [job spec](https://www.notion.so/tigrisdata/Senior-Developer-Advocate-3c13a312b52f4f5abb6cfef0c6522b63) contains my email, so please get in touch if you'd like to apply or if you've got any questions.

[![Senior Developer Advocate - Tigris Data](https://user-images.githubusercontent.com/328367/216959811-f8bc25fb-ce8f-4284-9431-c7663f3f68f8.png)](https://www.notion.so/tigrisdata/Senior-Developer-Advocate-3c13a312b52f4f5abb6cfef0c6522b63)

**Reply by InsaneNaman** [reply]
Hey @leggetter, I have sent you an email with all the required details. Looking forward to hear from you soon. 🧑‍💻

**Comment by Ronneback** [comment]
Enabler Mortgage | Full-Stack Engineer | Remote (Stockholm) | Part/Full-time | Equity base | https://thehub.io/jobs/63b43bd2b11234f717ca91b4

Take the chance to become a partner and for a more mature startup with permission from the Swedish Financial Supervisory Authority. Join us in building our platform for a brand new mortgage designed to give young adults the chance to enter the housing market. Enabler is originally from KTH Royal Institute of Technology and has developed an adapted forward-looking credit regulation system with machine learning that focuses on employability and potential instead of historical salary (like traditional banks) and can therefore lend a little more and with a lower own down payment. We have selected a handful of the country's biggest change makers who we would humbly and happily wish to have on the journey. 🏠 🏦 🚀

In your role, you will work in close collaboration with the founding team to, with your experience and iterative processes, essentially:

Further develop the platform

Further develop Custom endpoints and API Routes for the application

Data Fetching

Strapi CMS

Authentication with JWT (JSON Web Tokens)

Node/React/Next.js

For more information or questions please contact us at joachim@enabler.se

**Comment by fofiu** [comment]
[Monogram](http://monogram.io/) | Senior Next.js Engineer | Remote (North & South America) | Full-time | https://monogram.io/careers

**Introduction**

You don’t work at Monogram to remove roadblocks, you work here to pave new paths. You don’t just solve problems, you join a family of designers & developers who collaborate like clockwork, rise together, fall together, and do their life’s work here.

As part our strong, fast moving team that’s been a [Vercel partner](https://vercel.com/experts) from early days, you will be helping convert beautifully designed web applications into reality, including building performant APIs and modern front-ends using Next.js.

Your role also includes coordinating with the teams responsible for other layers of the application design & infrastructure.

**Duties**

- Collaborate on a daily basis with a team of product managers, engineers, and UI/UX designers to understand business requirements, user experience goals and pain points
- Ability to develop websites/web apps and backend APIs
- Have a strong understanding of Core Web Vitals and how code impacts performance
- Build reusable components and libraries for future use

**🚨 Required Skills**

- TypeScript
- Next.js / React / Preact
- Solid Node.js experience
- GraphQL & REST API concepts and implementation patterns
- Testing frameworks such as Jest, Playwright, etc.
- Auth0 and other authentication services
- Content modeling in CMS’es such as Contentful, Dato, Prismic, Sanity, etc.
- Quality/clean code
- 2+ years of experience working in a fast-paced development environment using React and any of the following (Svelte, Angular, Vue, etc.)

**💭 What’s not required**

- HTML/CSS perfection — usually HTML/CSS screens will be provided by a front-end developer focused solely on perfecting the UI in HTML/CSS

**🧁 Bonus**

- Svelte, SvelteKit
- Understanding of middleware concepts in Next.js
- A/B testing implementation
- PostgteSQL/RDBMS systems
- Understanding of API Security
- Stripe

**🍒 Even better**

- Python and some data science concepts
- Three.js

If interested, please send us an email with your resume (and/or a video introduction) to [careers@monogram.io](mailto:careers@monogram.io)

**Comment by IraES0116** [comment]
**EQUUS SOFTWARE | CLOUD SUPPORT ENGINEER | LISBON PORTUGAL | Full-time |  https://apply.workable.com/equus-software/j/B07B2CE8F3/**

Description
WHO WE ARE

Founded in 1999, Equus has continually been recognized by Inc. 5000 as one of the fastest growing companies in the United States. We continue to expand with offices in Denver, Dallas, Lisbon, London and Manila. Our relentless focus on creating innovative global mobility solutions has helped us become the market leader in our industry.

With over 300 clients around the world, we are committed to delivering the right solutions to our client’s problems. And we do this every day.

Our award-winning cloud-based technology has been recognized year-over-year for end-to-end functionality, efficiency savings and connected ecosystem. But we are more than just a technology company. We are experts in helping our clients get the right talent, in the right place, fast.

WHAT YOU DO

As a Cloud Support Engineer specializing in Azure and part of the IT Operations team, you will monitor, manage and maintain Equus’ client production environment. This role will heavily impact the growth and commitment of Equus Software to migrate clients from our existing cloud platform to Azure. You will be passionate about driving successful optimization within the environment. The opportunity to utilize the entire suite of Azure products within a global environment will allow you to develop SME level skills.

3-5 years monitoring cloud environments with an emphasis in Azure

Have a deep understanding of metric reporting – how to take information from Azure and create business cases to support environment optimization

Experience with Powershell scripting and SQL queries

Good communication skills

**No sponsorships available

NICE TO HAVE

Microsoft Azure certification

Database management experience

Strong Networking background

EDUCATION

Bachelor’s Degree in Information Technology or related field

Certifications in lieu of a degree will be considered

What is Equus?

Equus Software is a market leading Software Company specializing in Global Mobility Software. We provide award-winning service to our portfolio of high-profile global clients. Our team is known for providing creative software solutions to solve our clients’ complex business problems.

Why this is a great position:

Great salary and benefits – Highly competitive salaries, great bonuses for great contributors. We have enterprise-class benefits.

Great location – In the heart of Lisbon, with 290 days of sun a year and access to awesome restaurants and culture.

Great people – We only hire the best. That means you're working with the best.

Great culture – Nice office, relaxed and collaborative atmosphere, casual dress code.

Software is our business – This isn’t a department in a larger company. We’ve been developing software for 19 years, and we’re growing fast. You’ll be a part of a team that develops software quickly and effectively.

Effective development processes – We follow agile processes (kanban) while keeping focus on 2-month release plans. We also do continuous improvement.

Always learning something new – This is a large application that solves a lot of business problems for a lot of clients. You’ll develop a wide variety of skills, and you’ll probably become a subject matter expert in your first year.

Fostering innovation – Take advantage of Innovation Day with every release. Play with whatever technology you want, learn what you want, try out something new, and present it to the company the next day. Many innovations have made it into our products.

**Comment by fyffeandrew** [comment]
**Senior React Engineer | [Portugal](https://boards.greenhouse.io/getground/jobs/4789930004) or [London](https://boards.greenhouse.io/getground/jobs/4013839004) (hybrid working)**

Hello, we are GetGround. We are a well-funded 85 person Tech scaleup in Europe and Asia with a mission to make the world fairer and more productive. We are doing this by making assets more transparent, trustworthy and accessible - starting with real estate, the world's biggest asset class. Incomes have stagnated, so more access to wealth generating things like real estate = less wealth inequality. We are disruptors building a global network for trading illiquid assets - and we have first mover advantage!

GetGround in numbers:

- 🧑‍🤝‍🧑 85 colleagues (ex Google, Talkdesk, Facebook, Monzo, McKinsey...)
- 📈 Captured 10%+ of our addressable UK market already
- 🏘 £1Bn of property on our platform, in a $Trillion market
- 🌎 5 locations (UK, Portugal, Germany, HK, Singapore) 

You will join one of our cross functional Tech squads based in one of our Portugal hubs where you will work with product and design to create products that delight our users. This is an opportunity to join a well funded scale-up with a clear vision, purpose & mission.

The Engineering team is led by CTO and Co-Founder, Misrab, a Stanford Computer Scientist. You can read about his philosophy to product engineering -> https://medium.com/@GetGround

Some of the things we feel passionately about:

- Technical rigour through collaborative, multi-disciplinary and thorough refinement processes. We want hackers and painters!
- Simple, clear and concise communication and documentation of technical ideas
- A challenger mindset - to implement new patterns and libraries including improving readability and efficiency

**What you will do**

- Own the user-facing web frontend codebase of the application, currently built in React with Redux
- Own the internal web frontend application
- Design and write clean documentation
- Collaborate with Backend Engineers (who work with Golang) to integrate user-facing elements with server side logic
- Build reusable code and libraries for future use
- Mentor junior engineers

The experience you will have:

- React, Redux and Typescript
- ES6, HTML5, CSS & one statically typed language (e.g. C, Java, Go)
- Unit testing, integration testing, or similar testing frameworks
- Database technologies like SQL, noSQL, key-value stores (e.g. mySQL, Postgres, MongoDB, Redis...)

**Our benefits**

- 💰 Competitive salary + stock options
- 🏥 Private health + dental benefits
- 🏡 Flexible/hybrid working (check details with your recruiter)
- 🏖 12 mental health days off annually (1 per month) PLUS holidays and public days off
- 🧠 Mental health support - 24/7 access to mental health support, counselling, therapy & coaching through Spectrum.Life for all employees and their immediate family
- 🎳 Team and company-wide events
- 📈 360° performance reviews to promote a culture of growth and development
- 🎤 Support for conferences and professional learning & development

[+ other localised benefits depending on your office location e.g. free breakfast 🥣 daily)

**Comment by leerob** [comment]
> Killer opportunity for a Web Engineer to work on our marketing site, [http://watershed.com](https://t.co/FtJp7Dga0p) — built on Next.js on Vercel

https://twitter.com/spikebrehm/status/1629157091695886338

**Reply by hiddenmyth** [reply]
Hello
How are you?
Nice to meet you.
I felt strongly that your offer is a really good opportunity for my growth
I am a senior web developer and have 8 years experience in web development field
I have enough experience in Next and Vercel.
Hope to see you and discuss about that asap.
Kind regards.
dragon99steel@gmail.com

**Reply by gho11123** [reply]
Hello.
I am an energetic, creative, challengeable software engineer with over 7 years of experience in full-stack development and leading product cycles from conception to completion and I think this experience matches well with your requirement.
It would be a pleasure to start a new challenge at your company.
I'm available right now.
Please check my resume.
Best regards.
demonwizard023@gmail.com

**Reply by rodinaleksandr0219** [reply]
Hello. Thanks for sharing good opportunity for us.

I would like to discuss more details for this position with you.
My mail is rodnaleksandr0219@gmail.com

Best regards,

**Comment by sumitshinde-84** [comment]
Hello,
My name is Sumit Shinde, and I am a passionate and hardworking full-stack developer. I have recently completed my education and am actively seeking job opportunities to kickstart my career. While I don't have any professional experience yet, I have gained valuable skills and knowledge through my education and by contributing to open-source projects.
